flutter/lib/src/sentry_flutter.dart

```diff
@@ -53,7 +49,7 @@ mixin SentryFlutter {
 
     await Sentry.init(
       (options) async {
-        await optionsConfiguration(options);
+        await optionsConfiguration(options as SentryFlutterOptions);
```

Comment:
the compiler should have inferred automatically here :(

Reply:
@marandaneto Er actually no in flutter 2.0:

![image](https://user-images.githubusercontent.com/5236035/110202447-a6509500-7ea3-11eb-92a4-9f96107585fd.png)

Reply:
yep it was just a rant